docs/integrations/pytorch.rst

```diff
@@ -3,109 +3,370 @@
 Neptune-PyTorch Integration
 ===========================
 
-Neptune has implemented an integration with the PyTorch machine learning library.
+.. note::
```

Comment:
Like it as an idea. However, placing it at the very top suggest that it is most important - it looks like info banner.

If I landed here to learn about PyTorch, then it is not that crucial for me.

Unless, you did it to have half of the video visible at the first glance. If so, then cool :)

Reply:
Yeah, I do think that showing those higher-level libs (where you get more value with the integration) is very important. 

MLflow talks about integration with pytorch with explicit dependency of pytorch-lightning even.